crates/core/tedge_agent/src/file_transfer_server/mod.rs

```diff
@@ -1,4 +1,6 @@
+//TODO Rename this module to tedge_server
```

Comment:
I would prefer `http_server`, stressing this is about HTTP while tedge is obvious in the context of `tedge-agent`.

Reply:
Will be renamed as suggested, right before merging this PR.